config/samples/horizon_v1beta1_horizon.yaml

```diff
@@ -7,3 +7,11 @@ spec:
   secret: "osp-secret"
   customServiceConfig: |
     SESSION_TIMEOUT = 3600
+  memcachedInstance: horizon-memcached
+---
+apiVersion: memcached.openstack.org/v1beta1
```

Comment:
this should be later removed when install_yams deploy a separate memcached instance.

Reply:
We probably should merge https://github.com/openstack-k8s-operators/install_yamls/pull/338 first.